lib/ansible/modules/cloud/ovirt/ovirt_storage_domains.py

```diff
@@ -122,6 +122,11 @@
         description:
             - "If I(True) storage domain will be formatted after removing it from oVirt/RHV."
             - "This parameter is relevant only when C(state) is I(absent)."
+    discard_after_delete:
+        description:
+            - "If I(True) storage domain blocks will be discarded upon deletion. Enabled by default."
```

Comment:
Just note, that the code is shared also for file based stoarges, so what will happen if it will be sent for NFS for example? Won't it fail?

Reply:
Good catch :)